trading_ig/rest.py

```diff
@@ -510,20 +512,30 @@ def create_open_position(self, currency_code, direction, epic, expiry,
         else:
             raise IGException(response.text)
 
-    def update_open_position(self, limit_level, stop_level, deal_id,
-                             session=None):
+    def update_open_position(self, limit_level, stop_level,
+                             deal_id, trailing_stop=False,
+                             trailing_stop_distance=None,
+                             trailing_stop_increment=None, session=None):
         """Updates an OTC position"""
         params = {
             'limitLevel': limit_level,
-            'stopLevel': stop_level
+            'stopLevel': stop_level,
+            'trailingStop': False,
         }
+        if trailing_stop:
+            params = {
```

Comment:
exactly, it's my fault, i don't test this code. I re-write code step by step with comment without testing.

Reply:
It's ok to write code, without testing it =)))) it's a lot easier to read something and review/correct it. Do not feel like it's your fault. Your code is very good! And the idea and the addition are all good stuff! I have been travelling this weekend, otherwise, I probably would have merged your code (with small fix ups) by now =)